modules/logging-create-loki-cr-cli.adoc

```diff
@@ -49,15 +50,41 @@ It is not possible to change the number `1x` for the deployment size.
 <2> Specify the name of your log store secret.
 <3> Specify the type of your log store secret.
 <4> Specify the name of a storage class for temporary storage. For best performance, specify a storage class that allocates block storage. Available storage classes for your cluster can be listed by using the `oc get storageclasses` command.
-<5> LokiStack defaults to running in multi-tenant mode, which cannot be modified. One tenant is provided for each log type: audit, infrastructure, and application logs. This enables access control for individual users and user groups to different log streams.
---
+// end::pre-5.9[]
 
-. Apply the `LokiStack` CR by running the following command:
+// tag::5.9[]
 +
-[source,terminal]
+.Example `LokiStack` CR
+[source,yaml]
 ----
-$ oc apply -f <filename>.yaml
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki # <1>
+  namespace: openshift-logging
+spec:
+  size: 1x.small # <2>
+  storage:
+    schemas:
+      - effectiveDate: '2023-10-15'
+        version: v13
+    secret:
+      name: logging-loki-s3 # <3>
+      type: s3 # <4>
+      credentialMode: # <5>
+  storageClassName: <storage_class_name> # <6>
+  tenants:
+    mode: openshift-logging
 ----
+<1> Use the name `logging-loki`.
+<2> Specify the deployment size. In the {logging} 5.8 and later versions, the supported size options for production instances of Loki are `1x.extra-small`, `1x.small`, or `1x.medium`.
+<3> Specify the secret used for your log storage.
+<4> Specify the corresponding storage type.
+<5> Optional field, {logging} 5.9 and later. Supported user configured values are as follows: `static` is the default authentication mode available for all supported object storage types using credentials stored in a Secret. `token` for short-lived tokens retrieved from a credential source. In this mode the static configuration does not contain credentials needed for the object storage. Instead, they are generated during runtime using a service, which allows for shorter-lived credentials and much more granular control. This authentication mode is not supported for all object storage types.
```

Comment:
credentialMode can also have the value of 'token-cco' which is the default value when Loki is running on managed auth/STS mode on STS/WIF clusters.